ptx/sec_deriv_intro.ptx

```diff
@@ -1318,11 +1326,12 @@
           <!-- START figures/fig_piecewisecosx1.tex -->
           <image xml:id="img_piecewisecosx1" width="47%">
             <shortdescription>
-              Graph of derivative of function that is <m>\cos(x)</m> for values of <m>x \leq \pi/2</m> and <m>x \gt \pi/2</m>.
+              Graph of derivative of function.
             </shortdescription>
             <description>
               <p>
-                Graph of derivative of the function that exists at <m>\fp(\pi/2) </m> and follows a curve of <m>\cos(x)</m> for <m>x\leq\pi/2</m> and <m>0</m>.
+                Graph of derivative of the function that exists at <m>\fp(\pi/2)</m> and follows a curve of <m>\cos(x)</m> for <m>x\leq\pi/2</m> and <m>0</m> after that.
+                There is a sharp bend at <m>\pi/2</m>, hence it is not differentiable at that point but the point exists hence it is continuous.
```

Comment:
You need to use the word "it" sparingly, since it (I mean, "it") is ambiguous. Are you referring to f(x), or f'(x)?

What if you say something like:
```
From the graph, we can see that <m>\fp(x)</m> is defined, and even continuous, at <m>\pi/2</m>. However, there is a sharp bend in the graph of <m>\fp(x)</m> at that point, suggesting that <m>\fpp(\pi/2)</m> would not be defined.
```

Remember that you have room to write a whole paragraph (or even multiple paragraphs) in the long description.